osx/QMK Toolbox.xcodeproj/project.pbxproj

```diff
@@ -256,13 +296,23 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
```

Comment:
All of this noise should be in a separate PR for updating the Xcode project.

Reply:
Agrrerd.